Go/src/mqsamputils/connection.go

```diff
@@ -106,7 +123,7 @@ func CreateConnection(index int) (ibmmq.MQQueueManager, error) {
 	// The location of the KeyRepository is not specified in the CCDT, so regardless
 	// of whether a CCDT is being used, need to specify the KeyRepository location
 	// if it has been provided in the environment json settings.
-	if env.KeyRepository != "" {
+	if env.KeyRepository != "" && JwtCheck() == false {
```

Comment:
Re lines 75-89, if `JwtCheck() was true then would not be here, so the check for false, here is redundant.